src/com/franz/sud/components/engine/GameInitializer.java

```diff
@@ -213,16 +212,16 @@ public void setupUnits() {
                 .lifesteal(10)
                 .duration(5)
                 .build();
-        heal.setHeal(60);
-        lesserHeal.setHeal(30);
+        heal.setHealValue(60);
+        lesserHeal.setHealValue(30);
 
         // == Construction of units ==
 
         //Hero of the game
         SkilledUnit alucard = new SkilledUnit.Builder()

```

Comment:
I think you can just assign it directly to the `hero` variable.

Reply:
Have set the heal value in the constructor. It is already assigned during the construction of the unit. 
`SkilledUnit warlock = new SkilledUnit.Builder()
                .damage(25)
                .health(200)
                .name("Warlock")
                .evasion(10)
                .criticalChance(10)
                .lifesteal(25)
                .setSkill(heal)
                .build();`

Reply:
Hmm, I meant instead of doing this:
```
SkilledUnit alucard = new SkilledUnit.Builder()...build();
hero = alucard;
```
You can just do this:
```
hero = new SkilledUnit.Builder()...build();
```
Since there's no other purpose for the alucard variable.